inferentia/scripts/gen_triton_model.py

```diff
@@ -26,6 +26,61 @@
 
 import argparse
 import os
+from tensorflow.python.tools import saved_model_utils
```

Comment:
Move imports to after `if FLAGS.is_tensorflow_model:` so that pytorch model can be generated without it complaining of missing tf imports. 
We want users to gen pytorch inferentia models without having to install TF